conda_build/cli/main_convert.py

```diff
@@ -35,8 +37,8 @@
 """
 
 
-def parse_args(args):
-    p = ArgumentParser(
+def parse_args(args: Iterable[str] | None) -> tuple[ArgumentParser, Namespace]:
```

Comment:
In the other modules you used `args: Sequence[str]` for `parse_args`/`execute`.
`argparse.parse_args` should be fine with either (`argparse` does a `args = list(args)` internally).
Just wanted to point out the slight inconsistency

Reply:
@mbargull Since we're in a rush to get this out, could you work on a patch for this?

Reply:
Nevermind, I'll do it

Reply:
> Nevermind, I'll do it

Ah, didn't see that message.
I just pushed a change to use `Sequence[str]` here too.
Feel free to amend how you see fit.

Reply:
zing!

Reply:
This change is exactly I was about to push, no harm done :)

Reply:
Thank yoU!